code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -72,10 +90,24 @@
 			.()
 
 /obj/item/pinpointer/proc/workdisk()
-	scandisk()
-	point_at(the_disk, 0)
-	spawn(5)
-		.()
+	if(!mode)
+		scandisk()
+		point_at(the_disk, FALSE)
+		spawn(5)
+			.()
+
+/obj/item/pinpointer/proc/workbomb()
+	if(mode)
+		if(!syndicate)
+			scanbomb()
+			point_at(the_bomb, FALSE)
+			spawn(5)
+				.()
+		else
+			scanbomb()
+			point_at(the_s_bomb, FALSE)
+			spawn(5)
+				.()
```

Comment:
```suggestion
		var/to_search
		if(syndicate)
			to_search = the_s_bomb
		else
			to_search = the_bomb
		scanbomb()
		point_at(to_search, FALSE)
		spawn(5)
			.()
```

Reply:
These spawns and recursions are giving me bad vibes. I would rather pinpointers be assigned to something like `SSfastprocess`

Reply:
moved to SSfastprocess.